### ISSUE TITLE
Missing support for importlib_metadata v 4.0.1

### ISSUE THREAD

**Issue by Duky87** [jodal/pykka#131]
Refer to entry: https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/1400

While installing RPi-Jukebox-RFID on my Raspberry following error was shown:

```
pykka 3.0.0 has requirement importlib_metadata<4.0,>=1.6; python_version < "3.8", but you'll have importlib-metadata 4.0.1 which is incompatible.
```

pykka is not supporting the newest  importlib-metadata https://github.com/python/importlib_metadata

```
journalctl -u mopidy
-- Logs begin at Mon 2021-05-03 23:26:57 BST, end at Mon 2021-05-03 23:28:21 BST. --
May 03 23:27:09 raspberrypi systemd[1]: Starting Mopidy music server...
May 03 23:27:09 raspberrypi systemd[1]: Started Mopidy music server.
May 03 23:27:14 raspberrypi mopidy[3080]: Traceback (most recent call last):
May 03 23:27:14 raspberrypi mopidy[3080]: File "/usr/local/lib/python3.7/dist-packages/pkg_resources/init.py", line 568, in _build_master
May 03 23:27:14 raspberrypi mopidy[3080]: ws.require(requires)
May 03 23:27:14 raspberrypi mopidy[3080]: File "/usr/local/lib/python3.7/dist-packages/pkg_resources/init.py", line 886, in require
May 03 23:27:14 raspberrypi mopidy[3080]: needed = self.resolve(parse_requirements(requirements))
May 03 23:27:14 raspberrypi mopidy[3080]: File "/usr/local/lib/python3.7/dist-packages/pkg_resources/init.py", line 777, in resolve
May 03 23:27:14 raspberrypi mopidy[3080]: raise VersionConflict(dist, req).with_context(dependent_req)
May 03 23:27:14 raspberrypi mopidy[3080]: pkg_resources.ContextualVersionConflict: (importlib-metadata 4.0.1 (/usr/local/lib/python3.7/dist-packages), Requirement.parse('importlib_metadata<4.0,>=1.6; python_version < "3.8"'), {'Pykka'})
May 03 23:27:14 raspberrypi mopidy[3080]: During handling of the above exception, another exception occurred:
May 03 23:27:14 raspberrypi mopidy[3080]: Traceback (most recent call last):
May 03 23:27:14 raspberrypi mopidy[3080]: File "/usr/bin/mopidy", line 6, in
May 03 23:27:14 raspberrypi mopidy[3080]: from pkg_resources import load_entry_point
May 03 23:27:14 raspberrypi mopidy[3080]: File "/usr/local/lib/python3.7/dist-packages/pkg_resources/init.py", line 3242, in
May 03 23:27:14 raspberrypi mopidy[3080]: @_call_aside
May 03 23:27:14 raspberrypi mopidy[3080]: File "/usr/local/lib/python3.7/dist-packages/pkg_resources/init.py", line 3226, in _call_aside
May 03 23:27:14 raspberrypi mopidy[3080]: f(*args, **kwargs)
May 03 23:27:14 raspberrypi mopidy[3080]: File "/usr/local/lib/python3.7/dist-packages/pkg_resources/init.py", line 3255, in _initialize_master_working_set
May 03 23:27:14 raspberrypi mopidy[3080]: working_set = WorkingSet._build_master()
May 03 23:27:14 raspberrypi mopidy[3080]: File "/usr/local/lib/python3.7/dist-packages/pkg_resources/init.py", line 570, in _build_master
May 03 23:27:14 raspberrypi mopidy[3080]: return cls._build_from_requirements(requires)
May 03 23:27:14 raspberrypi mopidy[3080]: File "/usr/local/lib/python3.7/dist-packages/pkg_resources/init.py", line 583, in _build_from_requirements
May 03 23:27:14 raspberrypi mopidy[3080]: dists = ws.resolve(reqs, Environment())
May 03 23:27:14 raspberrypi mopidy[3080]: File "/usr/local/lib/python3.7/dist-packages/pkg_resources/init.py", line 772, in resolve
May 03 23:27:14 raspberrypi mopidy[3080]: raise DistributionNotFound(req, requirers)
May 03 23:27:14 raspberrypi mopidy[3080]: pkg_resources.DistributionNotFound: The 'importlib_metadata<4.0,>=1.6; python_version < "3.8"' distribution was not found and is required by Pykka
May 03 23:27:14 raspberrypi systemd[1]: mopidy.service: Main process exited, code=exited, status=1/FAILURE
May 03 23:27:14 raspberrypi systemd[1]: mopidy.service: Failed with result 'exit-code'.
```

My Python Version is:
```
pi@raspberrypi:~ $ python --version
Python 3.7.3
```

Base image and version
```
cat /etc/os-release
PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"
```

![Issue](https://user-images.githubusercontent.com/83616768/116994682-53dff900-acd9-11eb-98e4-9053680b4140.jpg)

**Comment by jodal** [comment]
Looks like this have already been fixed by Dependabot in commit 12f70611fe31584581cfcf8bf15f69fd74442474 and 23340fc680cdce0c5fa26760a99e46d5f4bc380a. I'll roll a new release ASAP!

**Comment by jodal** [comment]
Pykka 3.0.1 is on PyPI now. It should fix the incompatibility with importlib-metadata 4.0.1.
